docs/eBpfExtensions.md

```diff
@@ -373,15 +401,15 @@ To operate on the eBPF store, the user mode application needs to link with eBPFA
         _In_reads_(section_info_count) const ebpf_program_section_info_t* section_info, uint32_t section_info_count);
     ```
 
-- `ebpf_store_update_program_information`: updates program information in the eBPF store, given a pointer to an array of program information (i.e., `_ebpf_program_info`):
+- `ebpf_store_update_program_data`: updates program information in the eBPF store, given a pointer to an array of program information (i.e., `_ebpf_program_info`):
 
     ```c
     ebpf_result_t
-    ebpf_store_update_program_information(
+    ebpf_store_update_program_data(
```

Comment:
nit: since this API takes a pointer to `ebpf_program_info_t`, should the name of API also be aligned with similar name, i.e. `ebpf_store_update_program_information` instead of `ebpf_store_update_program_data`?

Reply:
renamed.